camptix.php

```diff
@@ -1778,17 +1778,16 @@ function field_enable_refunds( $args ) {
 		<?php
 	}
 
-	function field_enable_select2( $args ) {
-	    ?>
-        <div id="tix-select2-enabled-radios">
-            <label class="tix-yes-no description"><input type="radio" name="<?php echo esc_attr( $args['name'] ); ?>" value="1" <?php checked( $args['value'], true ); ?>> <?php _e( 'Yes', 'camptix' ); ?></label>
-            <label class="tix-yes-no description"><input type="radio" name="<?php echo esc_attr( $args['name'] ); ?>" value="0" <?php checked( $args['value'], false ); ?>> <?php _e( 'No', 'camptix' ); ?></label>
-        </div>
+	function field_enable_select2( $args ) {?>
```

Comment:
Kudos for doing coding standards fixes, and for doing them in a separate commit :)

The `Development Practices` doc has some helpful points on writing commit messages. 

> `Use the imperative mood when possible: “Relax term ID comparisons…” instead of “Relaxes term ID comparisons…”` (from Core handbook)

Something like `Replace spaces with tabs`, or just a generic `Apply coding standards` would fit a little bit better.

Reply:
It's probably best to avoid updating the commit messages, rebasing, and force-pushing until we've finished reviewing everything else, though, since the force-push will wipe out a lot of the GitHub comments.